mercadopago.php

```diff
@@ -76,8 +83,34 @@ public function __construct()
         $this->confirmUninstall = $this->l('Are you sure you want to uninstall the module?');
         $this->module_key = '4380f33bbe84e7899aacb0b7a601376f';
         $this->ps_versions_compliancy = array('min' => '1.6', 'max' => _PS_VERSION_);
+        $this->path = $this->_path;
```

Comment:
$this->path só está sendo utilizado nessa linha.

Reply:
includes/module/model/StandardCheckout.php:61
includes/module/model/TicketCheckout.php:61
includes/module/model/CustomCheckout.php:62